org/mozilla/jss/pkcs11/PK11Signature.java

```diff
@@ -317,12 +334,47 @@ native protected boolean engineVerifyNative(byte[] sigBytes)
     public void engineSetParameter(AlgorithmParameterSpec params)
         throws InvalidAlgorithmParameterException, TokenException
     {
-        Assert.notYetImplemented("PK11Signature.engineSetParameter");
+        //For now we only care about RSAPSS
+        if( isRSAPSSAlgorithm((SignatureAlgorithm) algorithm) ) {
+            digestAlgorithm = getRSAPSSDigestAlgFromSpec((PSSParameterSpec) params);
+        }
```

Comment:
Do we want to leave an else block and say it isn't implemented / supported to pass parameters to other signature schemes?

Reply:
Done, throw exception.